angular-ngrx-scss/src/app/shared/components/repo-card/repo-card.component.html

```diff
@@ -0,0 +1,27 @@
+<ng-container *ngIf="repo">
+  <h3 class="name-container">
+    <a class="name" href="/{{ repo.owner.login }}/{{ repo.name }}">{{
```

Comment:
We can place the logic (for example url composing logic) into component code and just bind it. Otherwise the template has alot responsibilities. I call this as Single Responsibility Principle.  From my point of view the template should only display the data.